documentation/docs/steps/abapEnvironmentPullGitRepo.md

```diff
@@ -20,7 +20,7 @@ In the first example, the host and the credentialsId of the Communication Arrang
 ```groovy
 abapEnvironmentPullGitRepo (
   script: this,
-  repositoryName: '/DMO/GIT_REPOSITORY',
+  repositoryNames: ['/DMO/GIT_REPOSITORY'],
   credentialsId: 'abapCredentialsId',
   host: '1234-abcd-5678-efgh-ijk.abap.eu10.hana.ondemand.com'
```

Comment:
Maybe remove the real host config from ABAP system and change to `host: 'test.yourABAPinstance.com'` or something else

Reply:
This is not a real host :-). But it is inspired by a real host to keep the example as similar to reality as possible